packages/sdk/src/dataProtector/sharing/createCollection.ts

```diff
@@ -1,28 +1,10 @@
-import { ethers, type Contract } from 'ethers';
-import { ABI as sharingABI } from '../../contracts/sharingAbi.js';
 import { WorkflowError } from '../../utils/errors.js';
```

Comment:
🟢 🟢 The file that is the most simplified.
- Less imports
- Less parameters for the function
- Less logic to instantiate the smart-contract through ethers

→ All done by the new `getCollectionContrat` helper.